sanity-io/structured-content-2022
# Fetch & list Sanity Program details, delete content breaking CI

# Fetch & list Sanity Program details, delete content breaking CI

## Intent

- Deletes /speakers page that seems to be unused both in approved and "In Review" Figma sketches. Fixes CI after d9682ed4f41c72bb8094821ee4cc04c08ce91d3c.
- Deletes all contents on ad-hoc /program page, replacing it with an attempt at copying the accordion component under "Program Schedules" in the Figma sketch

## Details

There are still some things not clear to me (e.g. Sessions having `duration` fields in Sanity vs. from->to times expected by Figma sketches) but I suspect that is due to the modeling process [still being WIP](https://wja.slack.com/archives/C02UKELNTE0/p1645572982557739). 

## Testing this PR

1. Open [/program](https://structured-content-2022-web-37shabol3.sanity.build/program)
2. Verify that an accordion with some content matching the [Programs](https://structured-content-2022-studio-n4hn2lwwa.sanity.build/desk/program;f4f26e84-4860-402e-9bcc-a5f1f5e610f1) Sanity content is displayed
3. Verify that CI once again passes
